uint8 db_1_flag   # 挡板 1挡起 0放下
uint8 db_2_flag   # 挡板 1挡起 0放下
uint8 db_3_flag   # 挡板 1挡起 0放下
uint8 db_4_flag   # 挡板 1挡起 0放下
uint8 tg_flag   # 推杆 1推出 0缩回
uint8 tp_flag   # 托盘 1/2/3/4号垃圾桶朝向